paper-toast.html

```diff
@@ -126,6 +124,36 @@
 
         properties: {
           /**
+           * The element to fit `this` into.
+           * Overridden from `Polymer.IronFitBehavior`.
+           */
+          fitInto: {
```

Comment:
Do you want people to override this? If not, you could just set it to `window` in `ready` or something